kiali/README.md

```diff
@@ -0,0 +1,85 @@
+# Install custom Kiali in Kyma
+
+## Overview
+
+The Kyma Kiali component brings limited configuration options in contrast to the upstream [`kiali-operator`](https://github.com/kiali/helm-charts/tree/master/kiali-operator) chart. Modifications might be reset at next upgrade cycle.
+
+An alternative can be a parallel installation of the upstream chart offering all customization options. This tutorial outlines how to achieve such installation in co-existent to the Kyma stack.
+
+## Prerequisites
+
+- Kyma as the target deployment environment.
+- kubectl > 1.22.x
+- helm 3.x
+
+## Installation
+
+### Preparation
+
+1. Export your Namespace as a variable. Replace the `{namespace}` placeholder in the following command and run it:
+
+    ```bash
+    export KYMA_KIALI_NS="{namespace}"
+    ```
+
+1. Export the Helm release name which you want to use. The release name has to be unique for the chosen namespace. Be aware that all resources in the cluster will be prefixed with that name. Replace the `{release-name}` placeholder in the following command and run it:
+    ```bash
+    export HELM_RELEASE_NAME="{release-name}"
+    ```
+
+1. Update your helm installation with the required helm repository:
+
+    ```bash
+    helm repo add kiali https://kiali.org/helm-charts
+    helm repo update
+    ```
+
+### Install the kiali-operator
+
+Kiali recommends to install Kiali via the kiali-operator always. So this tutorial will do this as well by using the kiali-operator helm chart.
+
+1. Run the Helm upgrade command which will install the chart if not present yet.
+    ```bash
+    helm upgrade --install --create-namespace -n ${KYMA_KIALI_NS} ${HELM_RELEASE_NAME} kiali/kiali-operator --set cr.spec.auth.strategy=anonymous -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/values.yaml
+    ```
+
+Hereby, use the [values.yaml](./values.yaml) provided with this tutorial which contains customized settings deviating from the default settings, or create your own one.
```

Comment:
```suggestion
You can either use the `[values.yaml](./values.yaml)` provided in this `kiali` folder, which contains customized settings deviating from the default settings, or create your own `values.yaml` file.
```

"which" is ambiguous, could refer to "the values.yaml" or "this tutorial", but in this case that might be acceptable.